NitroxClient/GameLogic/Helper/PlantableHelper.cs

```diff
@@ -0,0 +1,206 @@
+﻿using System;
+using UnityEngine;
+using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.Logger;
+using NitroxClient.MonoBehaviours;
+
+namespace NitroxClient.GameLogic.Helper
+{
+    public static class GameObjectExtensions
```

Comment:
We reserve extensions for only very common reusable logic.  Since items like 'FixPlantGrowth' are specific pieces of logic, it should go into a file specific to it.